config/locales/views/en.yml

```diff
@@ -1731,6 +1731,9 @@ en:
         update: Update Language
       support_available: Support available
     index:
+      language_code_abbr_title: language code
+      language_code_format: "(%{code})"
```

Comment:
These can be removed since they're no longer used. That should clear up the unused keys failure in `rspec ./spec/lib/i18n/i18n_tasks_spec.rb:15`.